plugin.video.digistorageplayer/resources/settings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <setting id="username" type="text" label="Username" value="" />
```

Comment:
Normally we require that labels used in settings.xml be translatable strings in a strings.po file. Because the labels you're currently using are self-explanatory you can leave as is. Please keep this in mind for the future.
Please see: http://kodi.wiki/view/Language_support#What_is_strings.po for more info.